### ISSUE TITLE
Cohort definition enhancement - ability to study multiple exposures in a single day

### ISSUE THREAD

**Issue by kmkostka** [OHDSI/Atlas#1980]
Cohort definitions currently use whole days as the unit of time. This means that the tool will only allow for one exposure per patient per day. 

The FDA BEST program is using the OMOP CDM for federated analysis. There is an interest in using ATLAS to design various cohort definitions of patients who receive transfusions as an exposure. In these cohort definitions, time must be smaller that one day. The questions being asked include: the number of transfusions per day per patient, or the number of units of blood transfused per episode per patient.

Today, this is not possible in ATLAS. It must be written as a database query. Given the OMOP CDM can fundamentally support time units smaller than one day, this ticket is raising the request for enhancing ATLAS as a tool to be able to so the same. This would help ensure cohort definitions created for multiple exposures in a single day are done so in a standard, reproducible way across the OHDSI network (e.g. not bespoke to a single study).

**Comment by pbr6cornell** [comment]
_DATETIME fields become required in OMOP CDM v6.0, so this will become a supported feature in ATLAS 3.0.   Until then, it may be useful to develop a standardized utility outside of ATLAS to perform these analysis, and that utility can then serve as an input design artifact to further flesh out this issue.

**Comment by chrisknoll** [comment]
Does the following logic work:

Entry event: the exposure to X.

Inclusion rule:  greater than 1 exposure to X between 0 days before and 0 days after (ie: > 1 exposure on the same date as the entry event)

not sure how _DATETIME fields work here unless you are saying you want to evaluate exposures being separated by hours instead of days?

**Comment by kmkostka** [comment]
Correct. This group is specifically looking at safety related to blood transfusions. Need to segment cohorts by a series of exposures separated by hours.

**Comment by chrisknoll** [comment]
Going to close this here, not that it is a bad idea, but it's not an Atlas function but a Circe function...and implementing hour-based criteria is something we need to think about how to incorporate that function into the API....and then we can expose that functionality in Atlas.

I undertand the need for by-hour analysis, but the focus so far has been multi-year longitudinal analysis.
